values/src/main/java/org/spine3/time/LocalDates.java

```diff
@@ -45,4 +46,22 @@ public static LocalDate today() {
                                           .build();
         return result;
     }
+
+    /**
+     * Obtains the date in the future after adding the specified amount of days.
+     */
+    public static LocalDate inDays(int daysDelta) {
```

Comment:
Please rename to `plusDays` as in `java.time`. Also rename the names of the parameter. Review other methods and name them after counterparts from `java.time`.